service/configs.py

```diff
@@ -0,0 +1,28 @@
+# System configurations
```

Comment:
The quality setting is mutually exclusive. For future projects, we can consider using configuration files or flags to remove the need for manual commenting. 

```suggestion
# To use a different quality setting, you need to comment out the current setting, and then uncomment the desired setting.
# System configurations
```